README.md

```diff
@@ -23,23 +23,34 @@ Connecting those modules with each other lets you create immensly useful agents.
 
 ## Installation
 
-Beehive requires Go 1.8 or higher. Make sure you have a working Go environment. See the [install instructions](http://golang.org/doc/install.html).
+Beehive requires Go 1.8 or higher. Make sure you have a working Go environment.
+See the [installation instructions of Go](http://golang.org/doc/install.html).
 
 ### From source
 
-The recommended way is to fetch the sources and run make. This requires [dep](https://github.com/golang/dep)
-to be installed on your system.
+The recommended way is to fetch the sources and run make.
+This requires [dep](https://github.com/golang/dep) to be installed on your system.
 
+If the `$GOPATH` environment variable is empty (`echo $GOPATH`), set it with:
+    
+    # Set the $GOPATH environment variable for the active shell
+    export GOPATH="/your/go/path"
```

Comment:
The most recent Go versions don't actually require you to set this env variable: if it's not set, it defaults to `$HOME/go`. Maybe we should consider that the default in the install instructions, too.

Reply:
@muesli this is false on my version of Debian ARM64 (the most recent). The user must explicitly set GOPATH. This also happens with Python and the other interpreters. This is good: for security reasons the *PATH variables should be explicitly set by the end user, possibly inheriting the defaults set explicitly by the administrator in compliance with SELinux configurations and other security systems.

Reply:
Which version of Go does it come with? Beehive depends on 1.8, the very version that made GOPATH optional and not required.

Reply:
> Which version of Go does it come with? Beehive depends on 1.8, the very version that made GOPATH optional and not required.

The version of the Debian package is: `2:1.10~5`.
The version of the Go interpreter is: `go version go1.10.4 linux/arm64`.
@muesli I am afraid that regardless of the version you will have to adapt to the concept that the environment variables are subject to security measures and must therefore be set explicitly. For example, in my system GOPATH must be set first in the shell during installation and then in the systemd service, because systemd isolates the program from the environment (always a good thing) and therefore you must always explicitly pass the environment variables. :-)
In this regard after you have accepted this push I will correct the systemd service file so that it can work without any further changes on all Debian systems. ;-)